enhancements/cluster-logging/multi-cluster-observability.md

```diff
@@ -0,0 +1,425 @@
+---
+title: multi-cluster-observability-addon
+authors:
+  - "@periklis"
+  - "@JoaoBraveCoding"
+  - "@pavolloffay"
+  - "@iblancasa"
+reviewers:
+  - "@alanconway"
+  - "@jcantrill"
+  - "@xperimental"
+  - "@jparker-rh"
+  - "@berenss"
+  - "@bjoydeep"
+approvers:
+  - "@alanconway"
+  - "@jcantrill"
+  - "@xperimental"
+api-approvers:
+  - "@alanconway"
+  - "@jcantrill"
+  - "@xperimental"
+creation-date: 2023-11-28
+last-updated: 2023-11-28
+tracking-link:
+  - https://issues.redhat.com/browse/OBSDA-356
+  - https://issues.redhat.com/browse/OBSDA-393
+  - https://issues.redhat.com/browse/LOG-4539
+see-also:
+  - None
+replaces:
+  - None
+superseded-by:
+  - None
+---
+
+# Multi-Cluster Observability Addon
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Multi-Cluster Observability has been an integrated concept in Red Hat Advanced Cluster Management (RHACM) since its inception but only incorporates one of the core signals, namely metrics, to manage fleets of OpenShift Container Platform (OCP) based clusters (See [RHACM Multi-Cluster-Observability-Operator (MCO)](rhacm-multi-cluster-observability)). The underlying architecture of RHACM observability consists of a set of observability components to collect a dedicated set of OCP metrics, visualizing them and alerting on fleet-relevant events. It is an optional but closed circuit system applied to RHACM managed fleets without any points of extensibility.
+
+This enhancement proposal seeks to bring a unified approach to collect and forward logs and traces from a fleet of OCP clusters based on the RHACM addon facility (See Open Cluster Management (OCM) [addon framework](ocm-addon-framework)) by enabling these signals events to land on third-party managed and centralized storage solutions (e.g. AWS Cloudwatch, Google Cloud Logging). The multi-cluster observability addon is an optional RHACM addon. It is a day two companion for MCO and does not necessarily share any resources/configuration with the latter. It provides a unified installation approach of required dependencies (e.g. operator subscriptions) and resources (custom resources, certificates, CA Bundles, configuration) on the managed clusters to collect and forward logs and traces. The addon's name is Multi Cluster Observability Addon (MCOA).
+
```

Comment:
Perhaps we should rethink the name MCO addon - because the word MCO and addon means certain things to ACM customers already. This would be very confusing for them. 
Aside of naming there needs to be a robust discussion around how many operators we should have. Do we have one for :
MCO - aka metrics, as we have today
MCL - for logging (just made up this name for clarity in this discussion)
MCT - for tracing  (just made up this name for clarity in this discussion)

or 
MCO MCO - aka metrics, as we have today
MCLT - for logging and tracing combined

or
MCO - aka metrics, as we have today
MCL - for logging
MCT - for tracing
+ One overall operator.

There are several pros and cons for each approaches. And as you can see, the impact is not only technical but also user experience related.